cinema/models.py

```diff
@@ -50,9 +50,11 @@
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
-    image = models.ImageField(null=True, upload_to=movie_image_file_path)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
+    image = models.ImageField(
+        blank=True, null=True, upload_to=movie_image_file_path
+    )
```

Comment:
The `image` field in the `Movie` model uses `ImageField` with `blank=True` and `null=True`, allowing it to be optional. The `upload_to` parameter is correctly set to use the `movie_image_file_path` function for determining the file path.